packages/plugins/i18n/server/constants/iso-locales.json

```diff
@@ -935,6 +935,14 @@
     "code":"ig-NG",
     "name":"Igbo (Nigeria) (ig-NG)"
   },
+  {
+    "code":"smn",
+    "name":"Inari Sami"
+  },
+  {
+    "code":"smn_FI",
```

Comment:
could you update the format from `languague_region` to `language-region` please? 🙂